pkg/resource/deploy/step.go

```diff
@@ -808,6 +943,10 @@ func (op StepOp) RawPrefix() string {
 		return "< "
 	case OpDiscardReplaced:
 		return "<<"
+	case OpImport:
+		return "= "
```

Comment:
any thought on this?  as opposed to maybe `<=` (to indicate we're sucking it in)?
(yes, this is bikeshedding to the nth degree).

Reply:
Honestly I’d like to use the `>` symbol that’s currently allocated to `Read`, but I prefer `=` otherwise.